### PR TITLE
185 contact info

### PR DESCRIPTION
**Pull request by christinach**
part of #185 

- [#185] Fix display of Contact info component in the docs
- Update display of contact info based on the selected theme'
- [#185] Fix Accessibility display in documents
- Update Accessibility display to toggle between different themes
- [#185] Include theme shade in footer theme calculation
- Fix copyright theme display

### CODE REVIEW

**Review by maxkadel — APPROVED** [review]
Thanks @christinach!

I'm also noticing some repetition between the themes in the different elements - would it be possible to factor these out / share them somehow? Or since we're letting folks use individual elements would that not work? (this is a non-blocking question, feel free to merge, just curious)

File: src/components/_LuxLibraryContactInfo.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <component :is="type" class="lux-library-contact" :class="theme">
+  <component :is="type" :class="['lux-library-contact', theme]">
```
Comment (sandbergja) [inline src/components/_LuxLibraryContactInfo.vue:2]: Nice!

File: src/components/_LuxLibraryContactInfo.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@
  */
 
 export default {
-  name: "LibraryContactInfo",
+  name: "LuxLibraryContactInfo",
```
Comment (sandbergja) [inline src/components/_LuxLibraryContactInfo.vue:21]: 👍🏻

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach !  🐬

**Comment by christinach** [comment]
> Thanks @christinach!
> 
> I'm also noticing some repetition between the themes in the different elements - would it be possible to factor these out / share them somehow? Or since we're letting folks use individual elements would that not work? (this is a non-blocking question, feel free to merge, just curious)

@maxkadel we can take a look as a future work but it is not exactly the same css switch. Some change links some change text color.

**Pull request merged**